pkg/common/common.go

```diff
@@ -329,3 +331,30 @@ var AsyncUploadFormPaths = []string{
 	UploadFormAsync,
 	"/v1alpha1/upload-form-async",
 }
+
+// VddkInfo holds VDDK version and connection information returned by an importer pod
+type VddkInfo struct {
+	Version string
+	Host    string
+}
+
+// TerminationMessage contains data to be serialized and used as the termination message of the importer.
+type TerminationMessage struct {
```

Comment:
This is a great idea, it could come in handy to transfer unexpected but non-error import info such as some odd content types. We had an attempt to improve the import behavior when importing html/text but the solution was a bit lackluster: https://github.com/kubevirt/containerized-data-importer/pull/2909. Just a comment though, that would be out of scope for the PR.

Reply:
Yes, exactly what I'm doing in #3103